### PR TITLE
code restructured

### PR DESCRIPTION
**Pull request by madhurajayashanka**

### CODE REVIEW

File: src/app/admin-views/accounts/accounts.component.html [hunk 1]
```diff
@@ -2,17 +2,44 @@
   <h2 class="title">Account Details</h2>
 
   <div class="card-row">
-    <mat-card routerLink="/admin/accounts/list/hr" class="card">
+
+
+      <mat-card *ngIf="pathParam==='main-company'" routerLink="/admin/accounts/list/company-hrs" class="card">
+        <mat-icon class="icon">person_pin</mat-icon>
+        <div class="title">Company Hrs</div>
```
Comment (ChameeraD) [inline src/app/admin-views/accounts/accounts.component.html]: Translation ?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -41,7 +50,13 @@ export class AppComponent implements OnInit, OnDestroy {
       } else {
         this.isPageNotFound = false;
       }
+      this.cdr.detectChanges(); // Trigger change detection
     });
+
+    // Hide the loading component after 3 seconds
+    setTimeout(() => {
```
Comment (ChameeraD) [inline src/app/app.component.ts]: This is not a good approach

File: src/app/employee-views/checkin-checkout/checkin-checkout.component.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  checkIn() {
+    if (this.withinCompanyRadius()) {
+      console.log('Check In button clicked');
+    }
+  }
+
+  checkOut() {
+    console.log('Check Out button clicked');
```
Comment (ChameeraD) [inline src/app/employee-views/checkin-checkout/checkin-checkout.component.ts]: remove all console logs

File: src/app/header/header.component.ts [hunk 4]
```diff
@@ -1,40 +1,67 @@
...
+    // Open the side overlay by calling the openSideOverlay method from the parent component
+    this.dialog.open(HeaderComponent, {
+      width: '250px', // Adjust the width as needed
+    });
+  }
+
+  openSettings(): void {
+    // Replace this with your actual settings logic
+    // For example, you can navigate to a settings page or show a modal
+    console.log('Settings button clicked');
```
Comment (ChameeraD) [inline src/app/header/header.component.ts:75]: ?

File: src/app/admin-views/accounts/accounts.component.html (hunk 1 above)
Comment (madhurajayashanka) [reply src/app/admin-views/accounts/accounts.component.html]: this is to verify if the logged in person is a main-company admin or a branch admin

File: src/app/app.component.ts (hunk 2 above)
Comment (madhurajayashanka) [reply src/app/app.component.ts]: I will fix this Aiye

**Pull request closed without merging**